sympy/stats/tests/test_joint_rv.py

```diff
@@ -53,6 +53,32 @@ def test_NormalGamma():
         3*sqrt(10)*gamma(S(7)/4)/(10*sqrt(pi)*gamma(S(5)/4))
     assert marginal_distribution(ng, y)(1) == exp(-S(1)/4)/128
 
+def test_MultivariateBeta():
+    from sympy.stats.joint_rv_types import MultivariateBeta
+    from sympy import gamma
+    mb = MultivariateBeta('B', [1, 2])
+    mb_c = MultivariateBeta('C', 1, 2)
+    a1_f, a2_f = symbols('a1, a2', positive = False)
```

Comment:
No spaces around `=` when specifying a keyword argument (here and in other places).